docs/guides/reverification.mdx

```diff
@@ -0,0 +1,205 @@
+      ```json
+      {
+        "clerk_error": {
+          "type": "forbidden",
+          "reason": "reverification-error"
+        }
+      }
+      ```
+  </Tab>
+</Tabs>
+
+### Handle reverification client-side
+
+After setting up reverification on the server-side, you must handle reverification on the client-side.
+
+The following example demonstrates how to use the [`useReverification()`](/docs/references/react/use-reverification) hook to detect authorization errors and automatically display a modal that allows the user to verify their identity. Upon successful verification, the previously failed request is automatically retried.
+
+<Tabs items={["Next.js Server Action", "Next.js Route Handler", "React"]}>
+  <Tab>
+    ```tsx {{ filename: '/app/perform-action/page.tsx' }}
+    'use client'
+
+    import { useReverification } from '@clerk/nextjs'
+    import { myAction } from '../actions'
+
+    export default function Page() {
+      const [performAction] = useReverification(myAction)
+
+      const handleClick = async () => {
+        const myData = await performAction()
+        // If `myData` is null, the user cancelled the reverification process
+        // You can choose how your app responds. This example returns null.
+        if (!myData) return
+      }
+
+      return <button onClick={handleClick}>Perform action</button>
+    }
+    ```
+  </Tab>
+
+  <Tab>
+    ```tsx {{ filename: '/app/transfer/page.tsx' }}
+    'use client'
+
+    import { useReverification } from '@clerk/nextjs'
+
+    export default function Page({ amount_in_cents }: { amount_in_cents: number }) {
+      const [transferMoney] = useReverification(
+        async () =>
+          await fetch('/api/reverification-example', {
+            method: 'POST',
+            body: JSON.stringify({ amount_in_cents }),
+          }),
+      )
+
+      return <button onClick={transferMoney}>Transfer</button>
+    }
+    ```
+  </Tab>
+
+  <Tab>
+    ```tsx {{ filename: '/src/components/TransferButton.js' }}
+    import { useReverification } from '@clerk/react'
+
+    export function TransferButton({ amount_in_cents }: { amount_in_cents: number }) {
+      const [transferMoney] = useReverification(() =>
+        fetch('/api/reverification-example', {
+          method: 'POST',
+          body: JSON.stringify({ amount_in_cents }),
+        }),
+      )
+
+      return <button onClick={transferMoney}>Transfer</button>
+    }
+    ```
+  </Tab>
+</Tabs>
+
+## Supported reverification configurations
+
+To define the time period of the reverification check, you can pass the one of the following configurations to the `has()` helper: `strict_mfa`, `strict`, `moderate`, and `lax`. See the [`has()` reference doc](/docs/references/nextjs/auth-object#check-authorization-params-with-custom-permissions) for more details.
```

Comment:
I believe us changing this part to navigate to another page, goes against @colinclerk 's [recommendation](https://github.com/clerk/clerk-docs/pull/1726#pullrequestreview-2453425723)

Reply:
ahh I see. @colinclerk can you take a look at how I've modified this section, and let me know if you're still adamant on repeating the table here?